cells/src/commonMain/kotlin/com/wire/kalium/cells/domain/usecase/RefreshCellAssetStateUseCase.kt

```diff
@@ -119,22 +120,38 @@ internal class RefreshCellAssetStateUseCaseImpl internal constructor(
     }
 
     private suspend fun refreshLocalData(node: CellNode) {
-        attachmentsRepository.getAttachment(node.uuid).map { attachment ->
-            if (attachment is CellAssetContent) {
 
-                if (attachment.contentHash != node.contentHash) {
-                    attachment.localPath?.let { fileSystem.delete(it.toPath()) }
+        val attachment = attachmentsRepository.getAttachment(node.uuid).getOrNull() as? CellAssetContent
+
```

Comment:
```suggestion
        val attachment = attachmentsRepository
        .getAttachment(node.uuid).getOrNull() as? CellAssetContent
            ?: return
```

Removing unnecessary indentation, and we don't need to wonder what happens after the let block :D